app/i18n/en/conf.php

```diff
@@ -126,13 +126,15 @@
 	),
 	'sharing' => array(
 		'_' => 'Sharing',
+		'add' => 'Add a sharing method',
```

Comment:
I think just `Add sharing method` would be better? Shall I change it while adding the Dutch translation?

Reply:
I do not mind either way. I think both forms are ok.

Reply:
It's also inconsistent with `Remove sharing method`. ;-)

Reply:
I went with ```Add a sharing method``` because it's undefined since you haven't choose it now, and with ```Remove sharing method``` because it's defined since you know which one you're deleting.
I don't mind if you want to change it.

Reply:
I'll take a look around at similar interfaces elsewhere.